narwhals/dependencies.py

```diff
@@ -72,7 +73,17 @@ def get_dask() -> Any:
 
 def get_dask_dataframe() -> Any:
     """Get dask.dataframe module (if already imported - else return None)."""
-    return sys.modules.get("dask.dataframe", None)
+    if "dask" in sys.modules:
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore",
+                message="\nDask dataframe query planning.*",
+                category=FutureWarning,
+            )
+            import dask.dataframe as dd
+
+        return dd
+    return None  # pragma: no cover
```

Comment:
could you revert this change please? that should fix the tox issue

Reply:
Yes! I need to dive into it, will finish tomorrow morning 🫡

Reply:
cool, thanks! it's only the highlighted change that needs reverting, the rest is fine 😇 

I suspect something went not quite right when resolving merge conflicts

no worries, this is how we learn 😄 🤗

Reply:
Yes sorry for that, I deleted somehow one of the parentheses while removing the markers from merge conflict, and when I was fixing it, I saw that I have unstaged changes to dependencies.py file,  I did add/commit only my changes but this file somehow ended up in the pull request, but that's prbably because of merging main into my branch (if I understand it correctly). I'm looking now how to proceed with reverting. I already clicked on the "revert" button. I see that it added a commit. Is this the same as doing "git revert -m 1 a9c34dd"

Reply:
😄 no worries at all (and no need to apologise)

I wouldn't do this automatically - git revert is for reverting the whole PR, rather than just a part of it

in this particular case, I ended up changing it as part of another PR, so nothing left to do 😇 

In the future, before merging, I'd suggest clicking "files changed" on the PR, refreshing the page, and then checking that there's nothing there which you didn't explicitly intend to add (if there is, it means something might not have gone right, e.g. when merging)